NebulaServer/Server.cs

```diff
@@ -13,6 +14,9 @@ namespace NebulaServer
 {
     public class Server : INetEventListener
     {
+        //TODO: Some sort of config file
+        public static String ServerIP = "192.168.88.31";
```

Comment:
Set it to 127.0.0.1 for now, I'll look into adding some configuration file for our server to deal with this.